cdcsdk-server/cdcsdk-server-s3/src/main/java/com/yugabyte/cdcsdk/sink/s3/config/S3SinkConnectorConfig.java

```diff
@@ -921,7 +921,9 @@ public String toString() {
     }
 
     public static void main(String[] args) {
+        // CHECKSTYLE:OFF
         System.out.println(getConfig().toEnrichedRst());
+        // CHECKSTYLE:ON
```

Comment:
Do these comments indicate something?

Reply:
Yes. Checkstyle is turned off for this line and then turned on again